apps/src/p5lab/AnimationTab/DeleteAnimationDialog.jsx

```diff
@@ -8,16 +8,29 @@ export default class DeleteAnimationDialog extends React.Component {
   static propTypes = {
     onDelete: PropTypes.func.isRequired,
     onCancel: PropTypes.func.isRequired,
-    isOpen: PropTypes.bool.isRequired
+    isOpen: PropTypes.bool.isRequired,
+    labType: PropTypes.string.isRequired
   };
 
   render() {
+    let assetType;
+    switch (this.props.labType) {
+      case 'GAMELAB':
```

Comment:
Updated. @madelynkasula is this what you mean?

Reply:
sorry, no, i meant use the P5LabType constant in the constants file:

https://github.com/code-dot-org/code-dot-org/blob/de3a71d53cfc6d160ef36527796eae1af858b7b0/apps/src/p5lab/constants.js#L8

so rather than `case 'GAMELAB'`, it would be `case P5LabType.GAMELAB`, which is the same end result but makes it easier to track usages of conditional logic based on lab type

Reply:
got it, thanks for clarifying!